presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/AddExchanges.java

```diff
@@ -642,10 +642,12 @@ private PlanWithProperties planTableScan(TableScanNode node, Expression predicat
 
         private Predicate<TableLayoutResult> layoutHasAllNeededOutputs(TableScanNode node)
         {
-            List<ColumnHandle> columnHandles = node.getOutputSymbols().stream()
+            List<ColumnHandle> nodeColumnHandles = node.getOutputSymbols().stream()
```

Comment:
The variable rename belongs to where the variable was introduced (`Prefer stream usage over Lists.transform in AddExchanges` commit)